backend/app/usecase/shortlink/updater_test.go

```diff
@@ -0,0 +1,225 @@
+// +build !integration all
+
+package shortlink
+
+import (
+	"testing"
+	"time"
+
+	"github.com/short-d/app/fw/assert"
+	"github.com/short-d/app/fw/timer"
+	"github.com/short-d/short/backend/app/entity"
+	"github.com/short-d/short/backend/app/usecase/keygen"
+	"github.com/short-d/short/backend/app/usecase/repository"
+	"github.com/short-d/short/backend/app/usecase/risk"
+	"github.com/short-d/short/backend/app/usecase/validator"
+)
+
+func TestShortLinkUpdaterPersist_UpdateShortLink(t *testing.T) {
+	t.Parallel()
+
+	now := time.Now().UTC()
+	alias := "boGp9w35"
+	validNewAlias := "eBJRJJty"
+	validNewLongLink := "https://httpbin.org/get?p1=v1"
```

Comment:
Let's embed them into the test case directly  so that we can update the tests independently in the future.

Reply:
https://github.com/short-d/short#test-structure